### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by Joshua-Nweze**
I created an individual translation file for each page, making it easier to edit and manage plus i added extra css.

### CODE REVIEW

File: assets/js/index_translate.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  ar.addEventListener("click", () => {
+    translate_btn.innerHTML = "Arabic";
+
+    publications.innerHTML = data.ar.publications;
+    talks.innerHTML = data.ar.talks;
+    hero_text.innerHTML = data.ar.hero_text;
+    mission_text.innerHTML = data.ar.mission_text;
+    mission.innerHTML = data.ar.mission;
+    highlighted_projects.innerHTML = data.ar.highlighted_projects;
+    masader.innerHTML = data.ar.masader;
```
Comment (zaidalyafeai) [inline assets/js/index_translate.js:27]: when you override the innerHTML content you are removing some stuff like links to the code, paper, and web. For example the masader `card-title` has the following 
```html
<h5 class="card-title" id="masader">Masader 
<a href="https://t.co/rGIpZGyWlJ" target="_blank" class="btn btn-color btn-sm right mr-2"><i class="fa fa-file"></i> Paper</a>
<a href="https://github.com/ARBML/masader" target="_blank" class="btn btn-color btn-sm right mr-2"><i class="fa fa-github"></i> Code</a>
<a href="https://arbml.github.io/masader/" target="_blank" class="btn btn-
</h5>
```

File: assets/js/index_translate.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  let data = await response.json();
+
+  let ar = document.querySelector("#ar");
+  let en = document.querySelector("#en");
+  let ar_style = document.getElementById("ar_style");
+
+  ar.addEventListener("click", () => {
+    translate_btn.innerHTML = "Arabic";
+
+    publications.innerHTML = data.ar.publications;
```
Comment (zaidalyafeai) [inline assets/js/index_translate.js:21]: Can't we do that in a for loop?

File: assets/js/index_translate.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    join_our_community.innerHTML = data.ar.join_our_community;
+    collaborators.innerHTML = data.ar.collaborators;
+
+    ar_style.classList.add("ar-text");
+  });
+
+  en.addEventListener("click", () => {
+    translate_btn.innerHTML = "English",
+
+    publications.innerHTML = data.en.publications;
```
Comment (zaidalyafeai) [inline assets/js/index_translate.js:68]: Same can't we do that in a for loop?

File: assets/js/index_translate.js (hunk 1 above)
Comment (zaidalyafeai) [reply assets/js/index_translate.js:27]: Same thing for News. It contains some hyperlinks which are overridden when you do the translation.

**Review by zaidalyafeai — CHANGES_REQUESTED** [review]

**Comment by Joshua-Nweze** [comment]
Can you send the translations for the words in the  hyperlinks.

**Comment by zaidalyafeai** [comment]
> Can you send the translations for the words in the hyperlinks.

Code: شفرة 
Web: موقع 
Paper: ورقة بحثية 
Here: هنا 
Post: مقال 

if there are any missing ones you can use google translate and I will fix that later if there are any issues.

**Comment by Joshua-Nweze** [comment]
Okay, thanks

**Pull request merged**